### PR TITLE
Add doc on AVX2 and Haswell

### PR DESCRIPTION
**Pull request by lluunn**
/cc @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/131)

### CODE REVIEW

**Comment by jlewi** [comment]
Can you add a note on the GKE getting started page in the troubleshooting page as well?

https://deploy-preview-131--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-gke/

It would be good to explain how users actually change the zone as well with GKE. I think they have to change the value of the ZONE environment variable.

File: content/docs/guides/troubleshooting.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ Linux:
 grep -ci avx /proc/cpuinfo
 ```
 
+### AVX2
+Some components requirement AVX2 for better performance, e.g. TF Serving.
+To ensure the nodes support AVX2, we added minCpuPlatform arg in our deployment
+[config](https://github.com/kubeflow/kubeflow/blob/master/scripts/gke/deployment_manager_configs/cluster.jinja#L105).
+
+This will fail in regions (e.g. us-central1-a) that do not have Intel Haswell.
```
Comment (jlewi) [inline content/docs/guides/troubleshooting.md]: "On GCP this will fail"

File: content/docs/guides/troubleshooting.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ Linux:
...
+In that case, please choose another region, or change the config to other 
+[platform](https://en.wikipedia.org/wiki/List_of_Intel_CPU_microarchitectures)
+newer than Haswell.
```
Comment (jlewi) [inline content/docs/guides/troubleshooting.md]: Are you saying that we won't  run in us-central1-a with Haswell; but if we pick a newer platform then we can run in us-central1a?

File: content/docs/guides/troubleshooting.md (hunk 1 above)
Comment (lluunn) [reply content/docs/guides/troubleshooting.md]: done

File: content/docs/guides/troubleshooting.md (hunk 2 above)
Comment (lluunn) [reply content/docs/guides/troubleshooting.md]: I think so. https://cloud.google.com/compute/docs/regions-zones/#available
us-central1-a has skylake which is newer than haswell

**Comment by lluunn** [comment]
Added some words in getting started with GKE. PTAL, thanks!

File: content/docs/guides/troubleshooting.md [hunk 3]
```diff
@@ -17,6 +17,16 @@ Linux:
...
 ```
 
+### AVX2
+Some components requirement AVX2 for better performance, e.g. TF Serving.
+To ensure the nodes support AVX2, we added minCpuPlatform arg in our deployment
+[config](https://github.com/kubeflow/kubeflow/blob/master/scripts/gke/deployment_manager_configs/cluster.jinja#L105).
+
+On GCP this will fail in regions (e.g. us-central1-a) that do not have Intel Haswell.
+In that case, please choose another region, or change the config to other 
+[platform](https://en.wikipedia.org/wiki/List_of_Intel_CPU_microarchitectures)
```
Comment (jlewi) [inline content/docs/guides/troubleshooting.md]: How will using a newer platform fix this issue? If a region doesn't have haswell but it has a newer platform then wouldn't it already get scheduled there?

I thought mincpuplatform can be satisfied by newer platforms.

Comment (lluunn) [reply content/docs/guides/troubleshooting.md]: No it needs to explicitly have that platform.

Comment (jlewi) [reply content/docs/guides/troubleshooting.md]: Lets add a link to 
https://cloud.google.com/compute/docs/instances/specify-min-cpu-platform
as well

**Pull request merged**